bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -734,10 +734,15 @@ Toggle to turn on no competition mode (only relevant if FATES is being used).
 Toggle to turn on FATES satellite phenology mode (only relevant if FATES is being used).                        
 </entry>
 
-<entry id="use_fates_logging" type="logical" category="physics"
-        group="clm_inparm" valid_values="" value=".false.">
-Toggle to turn on the logging module
-(Only relevant if FATES is on)
+<entry id="fates_harvest_mode" type="char*13" category="physics"
+        group="clm_inparm" valid_values="no_harvest,event_code,surfdata_file,luhdata_area,luhdata_mass">
+Set FATES harvesting mode by setting fates_harvest_mode to a valid string option.
+Allowed values are:
+ no_harvest: no fates harvesting of any kind
+ event_code: fates logging via fates logging event codes (see fates parameter file) only
+ surfdata_file: fates harvest driven by CLM landuse timeseries data (dynHarvestMod)
```

Comment:
By the way I really like the use of human readible strings here -- that really helps understanding and usability around this.

Reply:
addressed via https://github.com/ESCOMP/CTSM/pull/2507/commits/f985fbafafd5710e944d7715522966e042743361

Reply:
I ended up going with `landuse_timeseries` since I think that terminology will be more familiar to ctsm users and that is the specific file that it is pulling the harvest data from.